radiopropa/plugin-template/testPlugin.py

```diff
@@ -1,20 +1,20 @@
-import crpropa
+import radiopropa
 import myPlugin
 
 print("My Simulation\n")
 
-ml = crpropa.ModuleList()
+ml = radiopropa.ModuleList()
 
-ml.add(crpropa.SimplePropagation(1*crpropa.parsec, 100*crpropa.parsec))
-ml.add(crpropa.MaximumTrajectoryLength(1000*crpropa.parsec))
+ml.add(radiopropa.SimplePropagation(1*radiopropa.parsec, 100*radiopropa.parsec))
```

Comment:
Can parsec also be removed? Doesn't seem like a useful unit for RP

Reply:
I would leave it like this. It's just units. There are a lot of units which are not used but for completeness I would leave them in there. NuRadio has also all these units so.